admin/src/components/EditViewRightLinks/index.tsx

```diff
@@ -1,4 +1,3 @@
-import React from 'react';
 import { useCMEditViewDataManager } from '@strapi/helper-plugin';
```

Comment:
strapi/helper-plugin is deprecated - https://docs.strapi.io/dev-docs/migration/v4-to-v5/additional-resources/helper-plugin

Reply:
@Sagar-echo There is currently an [open PR to fix this](https://github.com/mattmilburn/strapi-plugin-preview-button/pull/140), but so far it has not been resolved

Reply:
> strapi/helper-plugin is deprecated - https://docs.strapi.io/dev-docs/migration/v4-to-v5/additional-resources/helper-plugin

is it deprecated? really? why?